lib/rules/no-top-level-side-effects.ts

```diff
@@ -6,24 +6,28 @@ import type {CallExpression, NewExpression, AssignmentExpression} from 'estree';
 import {IsCommonJs, isTopLevel} from '../helpers';
 
 type Options = {
+  readonly allowDerived: boolean;
   readonly allowedCalls: ReadonlyArray<string>;
   readonly allowedNews: ReadonlyArray<string>;
+  readonly allowFunctionProperties: boolean;
```

Comment:
This option should be set explicitly in the [preset configs](https://github.com/ericcornelissen/eslint-plugin-top/tree/e760ffab5f777ec26665f8f47daba6cc6eeafc2b/lib/configs).

Reply:
Resolved with cfd4465780362ebfe56af734be9e6abe549067de